control_plane/storcon_cli/src/main.rs

```diff
@@ -508,6 +513,29 @@ async fn main() -> anyhow::Result<()> {
                 })
                 .await?;
         }
+        Command::NodeAttached { node } => {
+            let describe_response = storcon_client
+                .dispatch::<(), NodeAttachedResponse>(
+                    Method::GET,
+                    format!("control/v1/node/{node}/attached"),
+                    None,
+                )
+                .await?;
+            let shards = describe_response.shards;
+            let mut table = comfy_table::Table::new();
+            table.set_header(["Shard", "Primary/Secondary"]);
```

Comment:
I wonder if we could have a third column here for whether the attached state has been observed or is just intended by the storage controller. Or maybe that's not useful, idk. What do you think?